src/utils.cc

```diff
@@ -14,9 +16,11 @@ namespace {
 #endif
 
 void objstore_memcheck(int64_t size) {
+#if defined(__unix__) || defined(__linux__)
   struct statvfs buffer;
   statvfs("/dev/shm/", &buffer);
   RAY_CHECK_LE(size + 100, buffer.f_bsize * buffer.f_bfree, "Not enough available memory for allocation by objstore"); 
```

Comment:
I'm not sure if these are necessary for `/dev/shm/`, but if it behaves like typical file systems, then:
- If `f_bavail` also works, use that instead of `f_bfree`.
- If applicable, check if `f_favail >= 1` as well.